README.md

```diff
@@ -3,25 +3,28 @@ This bundle provides a Mautic Housekeeping Command for database cleanup purposes
 
 ## Command
 
-Command to delete lead_event_log, campaign_lead_event_log, email_stats and email_stats_devices table entries. 
+Command to delete lead_event_log table entries, campaign_lead_event_log table entries, email_stats table entries that referenced emails entry is currently not published and email_stats_devices table entries.
+
+Important: If referenced email is ever switched back to published, the contacts will get the email again.
 
 ```
 bin/console mautic:leuchtfeuer:housekeeping
 ```
-By default, entries older than 365 days are deleted from the CampaignLeadEventLog, LeadEventLog, EmailStats and EmailStatsDevices tables.
+By default, entries older than 365 days are deleted from the CampaignLeadEventLog, LeadEventLog, EmailStats (only email_stats entries that referenced emails entry is currently not published) and EmailStatsDevices tables.
```

Comment:
Correction: "(only email_stats entries **where** the referenced email~~s~~ entry is currently not published)"

Reply:
Thanks @Moongazer for this spelling recommendation - implemented that with my current commit.